app-text/mupdf/mupdf-1.21.1.ebuild

```diff
@@ -154,7 +157,8 @@ src_install() {
 	fi
 
 	# Respect libdir (bug #734898)
-	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || die "Failed to sed pkgconfig file to respect libdir in src_install()"
+	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || \
+		die "Failed to sed pkgconfig file to respect libdir in src_install()"
```

Comment:
nit: the control operator is usually put on the next line
```suggestion
		|| die "Failed to sed pkgconfig file to respect libdir in src_install()"
```

Reply:
Makes sense, fixed.